tests/yam/agama/agama_cli.pm

```diff
@@ -0,0 +1,29 @@
+## Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run installation through CLI with Agama
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use strict;
+use warnings;
+use testapi;
```

Comment:
these two I think needs to be out from recent pr that already include this in base classes.

Reply:
The error https://openqa.suse.de/tests/18880484#details